pkgs/development/python-modules/mapbox-earcut/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, setuptools
+, numpy
+, pybind11
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mapbox-earcut";
+  version = "2.2.2";
```

Comment:
This version seems arbitrary.

Reply:
You're right. Something went wrong about this. `0.12.10` would be right. Will change that.